### ISSUE TITLE
Missing arrows to change tab on mac

### ISSUE THREAD

**Issue by impact27** [spyder-ide/spyder#13191]
### What steps will reproduce the problem?

1 - open enough tabs that they can't fit
2 - Notice the missing arrows
<img width="874" alt="Screenshot 2020-07-04 at 16 56 11" src="https://user-images.githubusercontent.com/6740194/86515212-b0535d00-be17-11ea-8432-b229dc9728bc.png">

## Versions

* Spyder version: 4.2.0.dev0 3df4cb6e0
* Python version: 3.7.7
* Qt version: 5.12.6
* PyQt5 version: 5.12.3
* Operating System: Darwin 19.5.0

### Dependencies

```

# Mandatory:
applaunchservices >=0.1.7      :  0.2.1 (OK)
atomicwrites >=1.2.0           :  1.3.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.2.2 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.10.2 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.1 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.7.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.6.5 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=1.0                   :  2.4.4 (OK)
pyls >=0.34.0;<0.35.0          :  0.34.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.6.0 (OK)
qtconsole >=4.6.0              :  4.8.0.dev0 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
sphinx >=0.6.6                 :  2.2.1 (OK)
spyder_kernels >=1.9.1;<1.10.0 :  1.10.0.dev0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  0.29.14 (OK)
matplotlib >=2.0.0             :  3.1.2 (OK)
numpy >=1.7                    :  1.17.4 (OK)
pandas >=0.13.1                :  0.25.3 (OK)
scipy >=0.17.0                 :  1.3.3 (OK)
sympy >=0.7.3                  :  1.4 (OK)
```

**Comment by ccordoba12** [comment]
@goanpeca, please take a look at this one asap because it's a regression introduced in 4.1.4.

**Comment by goanpeca** [comment]
I will see if the Qt version has anything to do, but this is what I get locally

<img width="717" alt="Screen Shot 2020-07-06 at 12 28 47" src="https://user-images.githubusercontent.com/3627835/86621700-40161a00-bf84-11ea-9613-2156f801fac8.png">

**Comment by impact27** [comment]
Maybe it has to do with the mac version? which one are you using?

**Comment by goanpeca** [comment]
This is probably related to some work I did @impact27. See https://github.com/spyder-ide/spyder/pull/13060

But, not sure what might be causing the issue. 

<img width="698" alt="Screen Shot 2020-07-06 at 13 30 43" src="https://user-images.githubusercontent.com/3627835/86626845-e6661d80-bf8c-11ea-91ce-fcd0fe774730.png">

**Comment by goanpeca** [comment]
Checking with Qt 5.12 (from conda-forge) it does indeed mess up things. So I will need to add a different CSS depending on the Qt version. However I DO see the arrows, the buttons are not placed correctly, but are there.


<img width="1181" alt="Screen Shot 2020-07-06 at 14 46 21" src="https://user-images.githubusercontent.com/3627835/86634430-77da8d00-bf97-11ea-8817-457b91c3cc63.png">

**Comment by impact27** [comment]
I think they are there for me as well, it's just that they are 1 px wide

**Comment by juanis2112** [comment]
Arrows look fine for me too, using macOS Catalina 10.15.3

**Comment by ccordoba12** [comment]
Reopening because this was not really fixed by PR #13203.
